src/main/java/org/mule/module/json/internal/JsonSchemaValidator.java

```diff
@@ -22,11 +24,14 @@
 import static com.google.common.base.Preconditions.checkState;
 import static com.google.common.collect.Lists.newArrayList;
 
+import com.github.fge.jackson.JsonLoader;
```

Comment:
Need to change the import order. 
The import order is this:

import static mule 
import static mulesoft
import static java
import static javax
import static others
import mule 
import mulesoft
import java
import javax
import others